### PR TITLE
:recycle: idk what TheUntraceable did here

### PR DESCRIPTION
**Pull request by Sigmanificient**
Signed-off-by: sigmanificient <edhyjox@gmail.com>

## Checklist

- [ ] - All code changes have been tested, if any
- [ ] - The documentation has been updated accordingly for any code changes
- [ ] - This PR fixes an already existing issue
- [ ] - This PR adds a new feature (new functions, methods, classes, parameters, etc.)
- [ ] - This PR fixes a bug
- [ ] - This PR is a breaking change (removes/renames functions, parameters etc.)
- [ ] - This PR is NOT a code change (changes Documentation files, README.md, CONTRIBUTING.md etc.)

### CODE REVIEW

File: EpikCord/abstract.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 from __future__ import annotations
 import asyncio
-from typing import Optional, List, TYPE_CHECKING, Union
+from typing import Optional, List, TYPE_CHECKING, Union, Any, Dict
```
Comment (TheUntraceable) [inline EpikCord/abstract.py:3]: Why

Comment (Sigmanificient) [reply EpikCord/abstract.py:3]: oops

File: EpikCord/abstract.py [hunk 2]
```diff
@@ -685,10 +687,10 @@ async def delete_followup(self):
 
 
 class BaseSlashCommandOption:
-    def __init__(self, *, name: str, description: str, required: Optional[bool] = None):
-        self.name: str = name
-        self.description: str = description
-        self.required: bool = required or False
+    def __init__(self, *, name: str, description: str, required: bool = None):
```
Comment (TheUntraceable) [inline EpikCord/abstract.py:690]: bool = None?

Comment (Sigmanificient) [reply EpikCord/abstract.py:690]: aaaaaaaaaaaaaaaaaaaa

Comment (Sigmanificient) [inline EpikCord/abstract.py:690]:
```suggestion
    def __init__(self, *, name: str, description: str, required: bool = False):
```

**Review by TheUntraceable — APPROVED** [review]
Is MyPy happy though?

**Comment by Sigmanificient** [comment]
> Is MyPy happy though?

Still 167 errors, for sure

**Comment by TheUntraceable** [comment]
no like with that single line is it happy

**Comment by Sigmanificient** [comment]
> no like with that single line is it happy

yes

**Pull request merged**